scripts/zones/King_Ranperres_Tomb/DefaultActions.lua

```diff
@@ -1,6 +1,7 @@
 local ID = zones[xi.zone.KING_RANPERRES_TOMB]
 
 return {
-    ['_5a0']      = { messageSpecial = ID.text.HEAVY_DOOR },
-    ['Tombstone'] = { messageSpecial = ID.text.A_TOMB },
+    ['_5a0']            = { messageSpecial = ID.text.HEAVY_DOOR },
+    ['Tombstone']       = { messageSpecial = ID.text.A_TOMB },
```

Comment:
Does not look like it is in IDs.lua

Reply:
Doesn't seem to be a 'Tombstone' npc in KRT either...